lineman/app/components/user_avatar/user_avatar.scss

```diff
@@ -1,31 +1,50 @@
-.avatar-initials{
-  @include smallIcon;
+// .avatar-initials{
```

Comment:
Can we remove? Also I don't see a replacement for the user-avatar coordinator class?